sympy/sets/tests/test_sets.py

```diff
@@ -423,6 +423,8 @@ def test_contains():
     assert Interval(0, 2, True, True).contains(0) is S.false
     assert Interval(0, 2, True, True).contains(2) is S.false
 
+    assert Interval(0, 2) in Interval(0, 2) is S.false
```

Comment:
You can see [here](https://github.com/sympy/sympy/blob/master/sympy/sets/sets.py#L511), that it returns bool False instead of sympify False